BBallStatsTrack/src/com/example/bballstatstrack/models/gameevents/TurnoverEvent.java

```diff
@@ -0,0 +1,69 @@
+package com.example.bballstatstrack.models.gameevents;
+
+import com.example.bballstatstrack.models.Player;
+import com.example.bballstatstrack.models.Team;
+import com.example.bballstatstrack.models.gameevents.foulevents.OffensiveFoulEvent;
+
+public class TurnoverEvent extends GameEvent
+{
+    public static final String TURNOVER_TYPE = "turnoverType";
+
+    private TurnoverType mTurnoverType;
+
+    public TurnoverEvent( TurnoverType type, Player player, Team team )
+    {
+        super( EventType.TURNOVER, player, team );
+        mTurnoverType = type;
+    }
+
+    @Override
+    public void append( GameEvent appendedEvent )
+    {
+        if( mAppended != null )
+        {
+            mAppended.append( appendedEvent );
+            return;
+        }
+        if( appendedEvent instanceof StealEvent && mTurnoverType == TurnoverType.STEAL )
+        {
+            mAppended = appendedEvent;
+            return;
```

Comment:
Probably not needed po since the remaining lines beneath does nothing once the if condition for this is true.